.github/workflows/build_ubuntu.yml

```diff
@@ -0,0 +1,92 @@
+name: Build Ubuntu
```

Comment:
In general i'm not sure if we would like another build system joining the club. Might be better to propose this within another PR.
Personally i'm in favor of github actions over semaphore since github actions has linux, mac AND windows support whilst semaphore only has linux and mac. Also once https://github.com/performous/performous/pull/532 is merged you can use that `.semaphore/semaphore.yml` as baseline for all of our build needs including distribution!

Reply:
Oh, I am sorry. That merged accidentally. :-(

Reply:
Can I revert the 4 commits and push again?

Reply:
It'd be better to do a rebase where you remove those 4 commits. That's alright :)